test/JDBC/upgrade/15_2/schedule

```diff
@@ -266,6 +266,7 @@ BABEL-SP_SPROC_COLUMNS_100-dep
 BABEL-3000-dep
 Test-sp_helprole-dep
 Test-sp_helprolemember-dep
+TestDatetime-sql-before-15-3
```

Comment:
JDBC test showed that both `TestDatetime-sql` and `TestDatetime-sql-before-15-3` are passed in the latest.
```
TestDatetime-sql-vu-prepare:                           Passed! (678/40000ms OK)
TestDatetime-sql-vu-verify:                            Passed! (434/40000ms OK)
TestDatetime-sql-before-15-3-vu-prepare:               Passed! (670/40000ms OK)
TestDatetime-sql-before-15-3-vu-verify:                Passed! (720/40000ms OK)
```
Then, why do we need `TestDatetime-sql-before-15-3`? Can we just use `TestDatetime-sql` here?